src/ucs/profile/profile_defs.h

```diff
@@ -62,15 +62,16 @@ typedef enum {
  * Profile output file header
  */
 typedef struct ucs_profile_header {
-    uint32_t                 version;       /**< File format version */
-    char                     ucs_path[1024];/**< UCX library path*/
-    char                     cmdline[1024]; /**< Command line */
-    char                     hostname[64];  /**< Host name */
-    uint32_t                 pid;           /**< Process ID */
-    uint32_t                 mode;          /**< Bitmask of profiling modes */
-    uint32_t                 num_locations; /**< Number of locations in the file */
-    uint32_t                 num_threads;   /**< Number of threads in the file */
-    uint64_t                 one_second;    /**< How much time is one second on the sampled machine */
+    uint32_t                 version;               /**< File format version */
+    char                     ucs_path[1024];        /**< UCX library path*/
+    char                     cmdline[1024];         /**< Command line */
+    char                     env_variables[1024];   /**< UCX environment variables*/
```

Comment:
1. it means older parser could not read profile from newer version and vice versa?
 need to try make it compatible and update file format version accordingly
2. better allocate exactly as much space as needed since the list of env vars could be long/short in different cases